### PR TITLE
Nico - Added Create Schools Option to Frontend

### LINKED ISSUE #44: Frontend Create School Option
Make an option on that page to create a new school

### PR DESCRIPTION
**Pull request by nicoshinozaki**
In this PR, an option on that page to create a new school was made.

Now that the 'create' option is implemented, I am free to work on the 'edit' option, as the 'delete' option is already available after fixing SchoolIndexPage.js, it would be the last item of the EPIC

Deployed at: https://organic-nicoshinozaki-dev.dokku-10.cs.ucsb.edu

Closes #44

### CODE REVIEW

**Review by NevMan1 — DISMISSED** [review]
LGTM

File: frontend/src/main/pages/SchoolEditPage.js
Comment (ukyl) [inline frontend/src/main/pages/SchoolEditPage.js:1]: Is this a placeholder?

**Review by ukyl — DISMISSED** [review]
LGTM

**Review by pconrad — DISMISSED** [review]
Code LGTM.  Will test on dokku before merging.

**Pull request merged**